lib/class-wp-duotone-gutenberg.php

```diff
@@ -712,6 +712,12 @@ private static function enqueue_custom_filter( $filter_id, $duotone_selector, $f
 	 * @param string $filter_value     The filter CSS value. e.g. 'url(#wp-duotone-blue-orange)' or 'unset'.
 	 */
 	private static function enqueue_global_styles_preset( $filter_id, $duotone_selector, $filter_value ) {
+		if ( ! array_key_exists( $filter_id, self::$global_styles_presets ) ) {
+			/* translators: %s: duotone filter ID */
+			$error_message = sprintf( 'The duotone id "%s" is not registered in theme.json settings.', $filter_id );
```

Comment:
If you remove the `.` after `settings`, then the error message sounds more like a full sentence:

```Notice: The duotone id "wp-duotone-does-not-exist" is not registered in theme.json settings in /Users/ajlende/Documents/gutenberg/lib/class-wp-duotone-gutenberg.php on line 718`